test-engineering/contract-tests/README.md

```diff
@@ -17,9 +17,7 @@ the individual components of the suite.
 To run the contract tests locally, execute the following from the repository root:
 
 ```text
-docker compose \ 
--f test-engineering\contract-tests\docker-compose.yml \ 
-up --abort-on-container-exit --build
+u
```

Comment:
Guessing this is a typo? Probably restore what was there.

Reply:
eeek oops